tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -20,17 +20,32 @@ include(`platform/intel/dmic.m4')
 
 DEBUG_START
 
+define(CHANNELS, `4')
+
+define(DMIC_PCM_48k_ID, `99')
+define(DMIC_PIPELINE_48k_ID, `4')
+define(DMIC_DAI_LINK_48k_ID, `1')
+
```

Comment:
This is a kind of framework code..which can be used across platforms.
The dailink definitions can vary per platform and has to align with the id's in machine driver.Is it a good idea to have these definitions as a part of m4 itself?
statically assigning the id's in a generic piece of code, forces the developer to use the same link id's in machine driver and this is not explicit.